### ISSUE TITLE
[Effect Suggestion] 88 MPH!

### ISSUE THREAD

**Issue by TerrizX** [gta-chaos-mod/ChaosModV#1925]
**Duration**
1 minute 30 seconds

**Effect Description**
 Spawns that alien car Cletus gives Trevor, to which if you go 88mph, the effect of the Back To The Future car happens (with the fire streaks) and then time around the player slows down extremely. The player can still operate at normal speed and have effect on the world (so shoot people or whatever). To get out of the slowed down effect, the player has to drive the car again at 88MPH and it reverts back to normal time. Maybe faster if it's very easy to get to 88MPH to which you can then rename the effect to however fast the player has to go in order to go into this slowed down time.

**Comment by kolyaventuri** [comment]
I don't believe it's possible to slow down _just_ the world around the player, without also slowing down the player as well (like 0.5x / 0.2x gamespeed)

**Comment by TerrizX** [comment]
> I don't believe it's possible to slow down _just_ the world around the player, without also slowing down the player as well (like 0.5x / 0.2x gamespeed)

But theres mods like the Flash mod or Quicksilver mod that have achieved this. Maybe its gonna be a pain in the ass to do but from those mods, it is possible.

**Comment by TerrizX** [comment]
> Aah I was unaware there were mods that achieved that already. Pretty neat lookin'. Doesn't seem like any of them are open-source though, so I'm not sure how it was achieved. That being said, definitely worth looking into. This would be neat. Only thing I might change is not limiting it to controlled by the car? It may not be entirely obvious what the player needs to do to activate it.

I suppose. I was just thinking if that car was spawned next to the player and the effect occured, one would think to drive the car, and unknowingly they'll drive it fast enough to see the effect occur. But I guess maybe getting out of that state may not compute for some, but then they may just drive the car again and time reverts. The effect's name is 88MPH lol so you'd think people would make a connection. Maybe idk, I also thought it may be simpler for yall if only one car was drive-able when time is slowed down but who am I to say really, just an assumption. Feel free to change aspects if you feel it makes it easier for people to get
